src/main/java/core/basesyntax/Drawer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawer {
```

Comment:
```suggestion
public interface Drawable {
```
drawer - ящик в столе)